static/app/components/dashboards/issueWidgetQueriesForm.tsx

```diff
@@ -44,18 +50,14 @@ class IssueWidgetQueriesForm extends React.Component<Props, State> {
     };
   };
 
-  getFirstQueryError() {
+  getFirstQueryError(key: string) {
```

Comment:
Is this being used anywhere? Just noticed that there are no function calls that were changed to reflect this function signature change

Reply:
Not used, removed!